cmd/arrai/info.go

```diff
@@ -13,11 +14,11 @@ var infoCommand = &cli.Command{
 }
 
 func info(c *cli.Context) error {
-	fmt.Printf("Version    : %s\n", Version)
-	fmt.Printf("Git commit : %s\n", GitFullCommit)
-	fmt.Printf("Date       : %s\n", BuildDate)
-	fmt.Printf("OS/arch    : %s\n", BuildOS)
-	fmt.Printf("Go version : %s\n", GoVersion)
+	fmt.Printf("Version    : %s\n", strings.TrimSpace(Version))
+	fmt.Printf("Git commit : %s\n", strings.TrimSpace(GitFullCommit))
+	fmt.Printf("Date       : %s\n", strings.TrimSpace(BuildDate))
+	fmt.Printf("OS/arch    : %s\n", strings.TrimSpace(BuildOS))
+	fmt.Printf("Go version : %s\n", strings.TrimSpace(GoVersion))
```

Comment:
Nit: this has the feel of defensive programming. We're in control of the creation of these values, so we should be able to guarantee they don't have leading/trailing spaces, and thus simplify all downstream code that depends on them.